params/config.go

```diff
@@ -113,8 +113,8 @@ type ChainConfig struct {
 
 	SubnetEVMTimestamp *big.Int `json:"subnetEVMTimestamp,omitempty"` // A placeholder for the latest avalanche forks (nil = no fork, 0 = already activated)
 
-	FeeConfig          *FeeConfig `json:"feeConfig,omitempty"`
-	AllowFeeRecipients bool       `json:"allowFeeRecipients,omitempty"` // Allows fees to be collected by block builders.
+	FeeConfig          FeeConfig `json:"feeConfig,omitempty"`
```

Comment:
is `omitempty` valid here?

Reply:
No: https://www.sohamkamani.com/golang/omitempty/